fpdf/output.py

```diff
@@ -938,6 +1041,9 @@ def _add_image(self, info):
 
         return img_obj
 
+    def _add_icc_objs_from_output_intents(self):
```

Comment:
I think this method is not needed anymore

Reply:
🤔

Well, this `_add_icc_objs_from_output_intents()` method was introduced in this PR 😅

Reply:
Yes please 🙂